### PR TITLE
Additional Repl tool commands for new module testing, etc.

### PR DESCRIPTION
**Pull request by marklnichols**

### CODE REVIEW

File: chainweb.cabal [hunk 1]
```diff
@@ -743,6 +743,7 @@ executable cwtool
         , filepath
         , errors >= 2.3
         , exceptions >= 0.8
+        , extra >= 1.6
```
Comment (mightybyte) [inline chainweb.cabal]: Can we get rid of this dependency?  I'd rather not add someone's personal grab-bag of random stuff into our dependency tree.  I'm fine with inlining whatever function we're using.

Comment (marklnichols) [reply chainweb.cabal]: That one is Neil Mitchell's extra library -- it's very useful and not quite so random.  But I can if you want...

Comment (mightybyte) [reply chainweb.cabal]: Now that we've launched I think we should avoid new deps without a really compelling reason and this one doesn't quite seem to make the bar in my mind.

Comment (marklnichols) [reply chainweb.cabal]: I was just going to take it out, but I just noticed its already a dependency in quite a few modules.  My addition was just for the cwtool executable.

**Pull request merged**